### PR TITLE
Add Automata mainnet and ContextFree testnet support

### PR DESCRIPTION
**Pull request by RyuH1**
Applying for adding Automata Mainnet into Live Networks and ContextFree Testnet to the Test Networks.

Local Running Screenshot:
- Automata Mainnet:
Screenshot 2021-11-01 at 4 59 52 PM

- Automata ContextFree Testnet:
Screenshot 2021-11-01 at 4 59 30 PM

### CODE REVIEW

File: packages/apps-config/src/api/spec/index.ts [hunk 1]
```diff
@@ -220,7 +222,7 @@ const spec: Record<string, OverrideBundleDefinition> = {
   vodka,
   'web3games-node': web3games,
   'zcloak-network': zCloak,
-  zeitgeist: zeitgeist
+  zeitgeist: zeitgeist,
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/index.ts:225]:
```suggestion
  zeitgeist: zeitgeist
```

`yarn lint` will fix all the issues (or most of them), locally.

Comment (RyuH1) [reply packages/apps-config/src/api/spec/index.ts]: Thank you, fixed

**Comment by RyuH1** [comment]
Hi @jacogr, we have launched our mainnet and I have reflected the change into this PR as well. It will be appreciated if you can take a look again.

**Comment by jacogr** [comment]
Sorry, one small conflict snuck in - 2 a's in production in the same day. The rest looks fine.

**Comment by RyuH1** [comment]
> Sorry, one small conflict snuck in - 2 a's in production in the same day. The rest looks fine.

Hi @jacogr I've fixed the conflicts, please take a look, thanks a lot😊

**Comment by jacogr** [comment]
CI seems to be failing since it expects alphabetical ordering - see https://github.com/polkadot-js/apps/runs/4296675575?check_suite_focus=true#step:3:673

**Comment by RyuH1** [comment]
> CI seems to be failing since it expects alphabetical ordering - see https://github.com/polkadot-js/apps/runs/4296675575?check_suite_focus=true#step:3:673

Thanks for noticing, have fixed it

**Review by jacogr — APPROVED** [review]

**Pull request merged**